src/main/java/io/github/thebusybiscuit/slimefun4/core/guide/options/LearningAnimationOption.java

```diff
@@ -0,0 +1,67 @@
+package io.github.thebusybiscuit.slimefun4.core.guide.options;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.annotation.Nonnull;
+
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.cscorelib2.data.PersistentDataAPI;
+import io.github.thebusybiscuit.cscorelib2.item.CustomItem;
+import io.github.thebusybiscuit.slimefun4.api.SlimefunAddon;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+
+public class LearningAnimationOption implements SlimefunGuideOption<Boolean> {
```

Comment:
This class should be package-private.
It's also missing javadocs.

```suggestion
class LearningAnimationOption implements SlimefunGuideOption<Boolean> {
```